test/unit/algo/container/soa_vector.cpp

```diff
@@ -247,3 +248,110 @@ TTS_CASE("Check soa_vector::begin/end behavior")
   while(uv_begin != uv_end)
     TTS_EQUAL( eve::read(uv_begin++), uv.get(i++));
 }
+
+TTS_CASE("Check types")
+{
+  using T        = udt::grid2d;
+  using ap       = eve::aligned_ptr<int,       eve::fixed<64 / sizeof(int)>>;
```

Comment:
why not using the `cache_line_cardinal` thing ?

Reply:
It's in detail. I figured testing using detail is not ideal